apps/utils/lib/helpers/path_resolver.ex

```diff
@@ -25,7 +25,7 @@ defmodule Utils.Helpers.PathResolver do
   path from a file.
   """
   def static_dir(app) do
-    serve_local_static = System.get_env("SERVE_LOCAL_STATIC")
+    serve_local_static = System.get_env("SERVE_LOCAL_STATIC") || false
```

Comment:
I'm wondering, should the default value actually be `true` or `false`?

Reply:
No idea.

Reply:
It was false before, though. Normalize.to_boolean/1 changed, thats why we required to be explicit.

Reply:
Okay, let's leave it at `false` then! Thanks